backstop/tests/table.html

```diff
@@ -613,15 +613,14 @@ <h2 id="default">Default</h2>
                   />
                 </svg>
                 dolor.pdf
-                <!-- Use warning.svg from generic-icons package. -->
+                <!-- Use status-warning.svg from @itwin/itwinui-icons package. -->
                 <svg
-                  class="iui-status"
+                  class="iui-status iui-icons-default iui-warning"
```

Comment:
This just requires a huge refactor......Getting really bad

Reply:
We can revert it back to the broken state and fix in another PR if you'd rather.

Reply:
no, that part is fine.. Just additional class iui-status does not make sense here...